protocol/message_validator.go

```diff
@@ -163,6 +164,11 @@ func ValidateReceivedChatMessage(message *protobuf.ChatMessage, whisperTimestamp
 		return errors.New("text can't be empty")
 	}
 
+	const maxLength = 4096
```

Comment:
probably this should be better placed at the top of the file, and maybe given a slightly different name:
`const maxChatMessageTextLength = 4096`